great_expectations/validator/validation_graph.py

```diff
@@ -1,9 +1,21 @@
+import logging
```

Comment:
@alexsherstinsky before merging this can you enable type-checking for this module?
Previously, this module only had 2 out-standing typing issues.

https://github.com/great-expectations/great_expectations/blob/develop/pyproject.toml#L146
To enable type-checking you need to remove the line I've linked above.

Reply:
@Kilo59 I have done it and committed -- now trying to reopen this PR, which I accidentally closed.  Thanks.